src/formatDate.js

```diff
@@ -50,7 +50,33 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const [, , , oldSeparator] = fromFormat;
+  const [, , , newSeparator] = toFormat;
```

Comment:
```suggestion
  const oldSeparator = fromFormat[3];
  const newSeparator = toFormat[3];
```
Давай використаємо трошки зрозуміліший запис